addons/spreadsheet/static/tests/global_filters/global_filters_model_test.js

```diff
@@ -1143,6 +1143,25 @@ QUnit.module("spreadsheet > Global filters model", {}, () => {
         assert.equal(model.getters.getGlobalFilters()[0].label, "Arthouuuuuur");
     });
 
+    QUnit.test("Can undo-redo a MOVE_GLOBAL_FILTER", async function (assert) {
+        const model = await createModelWithDataSource();
+        addGlobalFilter(model, LAST_YEAR_GLOBAL_FILTER, {});
+        addGlobalFilter(model, THIS_YEAR_GLOBAL_FILTER, {});
+        addGlobalFilter(model, NEXT_YEAR_GLOBAL_FILTER, {});
+
+        const lastYearFilterId = LAST_YEAR_GLOBAL_FILTER.id;
+
+        moveGlobalFilter(model, lastYearFilterId, 1);
+        assert.deepEqual(model.getters.getGlobalFilters()[1].id, lastYearFilterId);
+
+        model.dispatch("REQUEST_UNDO");
```

Comment:
You can use `undo(model)`

Reply:
does this util exist in 17.0 ? I can't find it. Also the rest of the file uses request_undo and redo so I will keep them for consistance.